docker_mcp/core/transfer/archive.py

```
@@ -201,56 +201,66 @@ async def create_archive(
+                if result.returncode != 0:
+                    raise ArchiveError(f"Failed to create archive: {result.stderr}")
 
-        return archive_file
+                return archive_file
+        except TimeoutError:
+            logger.error(
+                "Archive creation timed out",
+                timeout_seconds=3600.0,
+                archive_name=archive_name,
+                volume_paths=volume_paths
+            )
```

Ruff (0.14.4)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)